CIP-????/README.md

```diff
@@ -0,0 +1,67 @@
+---
+CIP: 101
+Title: Integration of `ripemd-160` into Plutus
+Status: Proposed
+Category: Plutus
+Authors:
+  - Tomasz Rybarczy <tomasz.rybarczyk@iohk.io>
+Implementors:
```

Comment:
If memory serves, @michaelpj has recommended that implementors should be mentioned here by name (personally, not just the IOG Plutus Team as per the current Implementation Plan).

Reply:
I no longer work for IOG, so I don't know whether the IOG Plutus Team has agreed to take this on, but yeah, someone needs to! I think CIPs with no implementers just tend to disappoint people.

Reply:
@zliu41 I guess the first question for `Implementors:` is to establish whether IOG has this curve inclusion on their agenda as we first asked in https://github.com/cardano-foundation/CIPs/pull/826#pullrequestreview-2074631093?

Reply:
@rphair  so I'm happy to help with the implementation but I'm not able to cover it all so for sure some people will be added to the list as we go.

Reply:
OK @paluh then the standard form would be `Implementors: []` (the empty set) for now & I'm adding this change request to the comment above.